web/src/app/util/ClickableText.tsx

```diff
@@ -0,0 +1,50 @@
+import { makeStyles } from '@material-ui/core'
+import React from 'react'
+
+const karakurenai = '#cd1831'
+
+const useStyles = makeStyles({
+  button: {
+    color: karakurenai,
+    all: 'unset',
+    backgroundColor: 'transparent',
+    border: 'none',
+    cursor: 'pointer',
+    textDecoration: 'none',
+    display: 'inline',
+    margin: 0,
+    padding: 0,
+    '&:hover': {
+      textDecoration: 'underline',
+    },
+    '&:focus': {
+      textDecoration: 'underline',
+    },
+  },
+})
+
+interface ClickableTextProps {
+  text: string
+  onClick: () => void
+}
+
+// ClickableText
+// since anchor elements have become entrenched with navigation by convention
+// it is recommended that clickable text be implemented using a button element
+// see https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md
+function ClickableText({ text, onClick }: ClickableTextProps): JSX.Element {
+  const classes = useStyles()
+  return (
+    <button
+      onClick={(e) => {
+        e.preventDefault()
+        onClick()
+      }}
+      className={classes.button}
+    >
+      {text}
+    </button>
+  )
```

Comment:
We can just make the `onClick` a passthrough with other props -- that will make it usable for things like tooltips or things that require the event without needing to define them directly and any other props the caller might need (e.g. like `data-cy` or `aria-*`.

We could also use `children` instead of text, `<ClickableText>foobar</ClickableText>` but I think `<ClickableText text="foobar" />` is fine for it's intended use.

```suggestion
  const {text, ...rest} = props
  return (
    <button
      className={classes.button}
      {...rest}
    >
      {text}
    </button>
  )
```

Reply:
Missed this comment with my recent review, submit should only happen if `type=submit` -- and if that's set we shouldn't be overriding it. It's up to the renderer to decide if that's the behavior they want.

We have little need to require an `onClick` handler, as it's just a pass-through. It is a button, so we can probably assume there will be one, but if say the parent has the `onClick` handler instead of the button in some scenario and the button is just for visuals -- that shouldn't be a problem.

That said, if we want it explicitly in prop-types it should be defined as optional, passed a SyntheticEvent, etc.. so it follows the standard API.

---

The purpose of this component is just a link-styled button, so having it be a passthrough to a button that's pre-styled seems like it would be the minimal required API.